src/client/eppo-client.spec.ts

```diff
@@ -603,6 +607,216 @@ describe('EppoClient E2E test', () => {
     expect(assignment).toEqual('control');
   });
 
+  it('returns control variation and logs status_quo if subject is in holdout', () => {
+    const entry = {
+      ...mockExperimentConfig,
+      allocations: {
+        allocation1: {
+          percentExposure: 1,
+          holdout: {
```

Comment:
Here's how I'm thinking of formatting holdouts in the RAC

Reply:
Consider naming it `holdouts` as there can be more than one that apply

Reply:
Ah yeah, I was thinking we usually use plurals to indicate an iterable, e.g. `holdouts.map((holdout) => {do stuff with each holdout})` but that wouldn't apply here, so thinking the singular `holdout` makes sense to indicate the overall holdout configuration, which would include an iterable list of holdout `keys` (in plural) within

Reply:
oooh, this makes sense to me! I like that.